src/pages/share.hbs

```diff
@@ -17,12 +17,34 @@ headline: Share the joy of praying for others
 		<!-- Main -->
 			<section id="main" class="container">
 				<div class="box special">
-          <div class="infographic">
-            <span class="image fit"><img src="images/ceaseless_fb_infographic_en.jpg" alt="" /></span>
+          <h2><b>Help get the word out about Ceaseless!</b></h2>
+          <p>Below are some materials to help you share Ceaseless with your friends, family, church, and beyond.
+            <br/>
+            Hover over an image and pick which social network you would like to share it with.</p>
+          <div class="share shortcuts">
+            <h2>Overview</h2>
+            <ul>
+              <li><a href="#infographic">Infographic</a></li>
+              <li><a href="#banner">Banner</a></li>
+              <li><a href="#ceaseless-info-video">Video</a></li>
+            </ul>
+          <br/>
+          <div class="infographic" id="infographic">
+            <h2>Infographic</h2>
+            <span class="image fit"><img class="shareable-class" src="images/ceaseless_fb_infographic_en.jpg" alt="Ceaseless Prayer Infographic" /></span>
+          </div>
+          <br/>
+          <div id="banner">
+            <h2>Banner - Facebook/Twitter Profile Cover Image</h2>
+            <span class="image fit"><img class="shareable-class" src="images/ceaseless_android_lifestyle_shareable_banner.jpg" alt="Ceaseless Prayer - Pray for your world 3 friends at a time" /></span>
           </div>
           <br/>
-          <div class="video-container">
-            <iframe width="560" height="315" src="https://www.youtube.com/embed/1KUHsYbZz8k" frameborder="0" allowfullscreen></iframe>
+          <div>
+            <h2>Video - Personal Prayer for Everyone on Earth</h2>
```

Comment:
We don't want to overwhelm the user, but should we link to the app store video and my talk video too?